### ISSUE TITLE
ADOBE 2.0

### ISSUE THREAD

**Issue by library-user** [apprenticeharper/DeDRM_tools#618]
I have moved to a new PC and need to authorise the PC for AdobeID. The earliest version of Adobe I can find is version 3.0 and, according to your blog, this means that the DeDRM tools will not work on downloaded books as the encryption has changed. 
Is there anywhere I can still get hold of Version 2, please?
Thanks

**Comment by ElleKayEm** [comment]
Here's a direct link to the installer:

http://download.adobe.com/pub/adobe/digitaleditions/ADE_2.0_Installer.exe

**Comment by library-user** [comment]
Many thanks

**Comment by library-user** [comment]
OK. Here' where I am. My old computer is soon to be going in the bin so I have a new, Windows 10 one. On the old computer, I loaded Calibre version 1.48 because that computer runs Windows XP. I installed the latest DeDRM tools and all the books I have in Reader Library were de-DRM'd successfully. Obviously, they are in Calibre Library on the XP computer. I need these DeDRM tools to work on the new PC, so Idownloaded the lateset Calibre and installed the latest DeDRM plugin. However, when I import one of the books (which was previously successfully DeDRM'd on the old PC) the DeDRM fails. The log is attached. I did point Calibre at the file over a network share - but I can't see that that would make a difference. I have not yet created a new AdobeID on this new PC as I am waiting for Sony Customer Support to tell me how to deauthorise two PRS-350 readers - one for me and one for my wife.
Keeping the old PC system is not an option!
Hope you can help
Thanks
library-user
==================================================================
calibre Debug log
calibre 3.30 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.17134-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.17134')
Python 2.7.12+
Windows: ('10', '10.0.17134', 'SP0', u'Multiprocessor Free')
Interface language: en_GB
Successfully initialized third party plugins: DeDRM (6, 6, 1)
calibre 3.30 [64bit]  embedded-python: True is64bit: True
Windows-10-10.0.17134-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.17134')
Python 2.7.12+
Windows: ('10', '10.0.17134', 'SP0', u'Multiprocessor Free')
Interface language: en_GB
Successfully initialized third party plugins: DeDRM (6, 6, 1)
Turning on automatic hidpi scaling
devicePixelRatio: 1.0
logicalDpi: 120.0 x 120.0
physicalDpi: 108.917922948 x 108.857142857
Using calibre Qt style: True
[0.00] Starting up...
[0.00] Showing splash screen...
[0.06] splash screen shown
[0.06] Initializing db...
[0.06] db initialized
[0.06] Constructing main UI...
[1.33] main UI initialized...
[1.33] Hiding splash screen
[23.50] splash screen hidden
[23.50] Started up in 23.50 seconds with 12 books
stdout+stderr from file dialog helper: ['', '']
piped data from file dialog helper: ['\xd8\xb0\xf0\xfb\xfa\xf25\x11\x1bn\x8a\x19\xf9\xa1\x1d\xed\xa0\xb1\x07d\xcb\xe6 \x86?w\x04]\xabl@\x7f', '\\\\BRIAN-28CCA27D7\\MDocs Share\\My Books\\Reader Library\\This Boy.epub']
DeDRM v6.6.1: Trying to decrypt This Boy.epub
DeDRM v6.6.1: Verifying zip archive integrity
DeDRM v6.6.1: This Boy.epub is a secure Adobe Adept ePub
DeDRM v6.6.1: Looking for new default Adobe Digital Editions Keys after 3.7 seconds
DeDRM v6.6.1: Exception when getting default Adobe Key after 3.7 seconds
Traceback (most recent call last):
  File "calibre_plugins.dedrm.__init__", line 346, in ePubDecrypt
  File "calibre_plugins.dedrm.adobekey", line 377, in adeptkeys
ADEPTError: Adobe Digital Editions not activated
DeDRM v6.6.1: Ultimately failed to decrypt after 3.7 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Running file type plugin DeDRM failed with traceback:
Traceback (most recent call last):
  File "site-packages\calibre\customize\ui.py", line 171, in _run_filetype_plugins
  File "calibre_plugins.dedrm.__init__", line 632, in run
  File "calibre_plugins.dedrm.__init__", line 403, in ePubDecrypt
DeDRMError: DeDRM v6.6.1: Ultimately failed to decrypt after 3.7 seconds. Read the FAQs at Harper's repository: https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md
Added This Boy to db in: 0.2
Added 1 books in 4.5 seconds
Worker Launch took: 0.161999940872
==================================================================

**Comment by ElleKayEm** [comment]
For DeDRM to automatically get the key, Adobe Digital Editions authorized with your Adobe ID must be on the same computer and Windows user as calibre with DeDRM.  In this case, you could save your key from the DeDRM plugin on your old computer, then import it into the plugin on your new computer.

**Comment by library-user** [comment]
Hi, ElleKayEm
Thank you for your reply, but Im' afraid that I don't know how to perform the suggested actions.
1. Where is the 'key' and 2. how then do I import it into the plugin on the new computer.
A third question is that if I succeed in importing the key to the plugin on the new computer, will it then affect future actions when I finally get the eReader authorised on the new PC. (Waiting to here from Sony on this issue)
Thanks
library-user

**Comment by ElleKayEm** [comment]
If you go into the ADE section of the plugin customization, you should see a default key listed.  Select it and click the save button.  Name the file oldkey or whatever.  Then in the same section in your new setup, click the "import existing key file" button.  Since you can have more than one key, this shouldn't cause any problem with a new default key being created later from your new Adobe ID.

Of course, all of your old books have already had their DRM removed, right?  So you can just move the DRM-free copies from old computer to new.

**Comment by library-user** [comment]
Thanks for this. I've got it working. The key was called _default_key_ on the old PC, so I saved it in a file called _oldpc_key.der_ When I imported this into the new system, it showed up as _oldpc_key_ This didn't work, so after a few gos, I imported it and renamed it _default_key_ and everything then worked OK. 
Re your last para in your last help posting : Yes, all my books on the old PC have been DeDRM'd and are now in Calibre library on the new setup. The reason I'm going through this process is to check that the Calibre/DeDRM setup on the new PC will, in fact, DeDRM any new books I purchase. The only way I could think of to check that was to import a book that I knew had already been DeDRM'd successfully on the old system. Obviously, I imported the non DeDRM'd version from Reader Library for this test!
Many, many thanks for your help. I'm so glad that someone understands DRM which I think is a dreadful imposition on people who purchase a product but, it seems, do not 'properly' own it.
Thanks again
library-user
